docs/user/wallets/dashcore/advanced.rst

```diff
@@ -47,6 +47,31 @@ affecting the locked UTXOs.
 
 .. _dashcore-hd:
```

Comment:
This label belongs just prior to the "HD Wallets" heading. We should add a label here though for testnet

Reply:
Done.